mail-server/users.nix

```diff
@@ -55,16 +55,16 @@ let
     # is null, remove the file.
     ${lib.concatMapStringsSep "\n" ({ name, sieveScript }:
       if lib.isString sieveScript then ''
-        cat << EOF > "/var/sieve/${name}.sieve"
+        cat << EOF > "/var/sieve/${name}/default.sieve"
```

Comment:
I think we need a
`mkdir /var/sieve/${name}`
here

Reply:
Right, fixed, thanks.